package.json

```diff
@@ -22,6 +22,7 @@
     "react-native-gesture-handler": "~2.12.0",
     "react-native-safe-area-context": "4.6.3",
     "react-native-screens": "~3.22.0",
+    "react-native-svg": "^14.0.0",
```

Comment:
I'm getting a good amount of errors from this, I'm not sure if it's this package or not though.